packages/contracts/test/helpers/LiveTestHelper.sol

```diff
@@ -333,6 +334,8 @@ contract LiveTestHelper is Test {
             locked
         );
 
+        deal(address(dollarToken), address(staking), 100e18);
```

Comment:
This is kind of a hack because it sets the staking contract's balance out of nothing. I mean in a normal flow user should deposit tokens tokens for the staking balance to increase. I think it is clearer to increase staking contract's balance vie the `deposit` method. When a new dev opens the tests he wants to see the normal flow to understand the entire staking mechanic.

Reply:
okay. got it

Reply:
This is the only unresolved conversation. Looks like the fix was not implemented yet?

Reply:
its fixed